cyipopt/scipy_interface.py

```diff
@@ -118,7 +118,9 @@ def __init__(self,
             fun = MemoizeJac(fun)
             jac = fun.derivative
         elif not callable(jac):
-            raise NotImplementedError('jac has to be bool or a function')
+            raise NotImplementedError(
+                'jac has to be bool or a function (got {!r})'.format(type(jac))
+            )
```

Comment:
As CyIpopt is now Python>=3.7 can we please make the f-strings instead of using `.format()`?